spring-boot-security-taglibs/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1"%>
```

Comment:
I'm curious why you use a pageEncoding here of ISO-8859-1 while taking the time to set up the app for UTF-8?